<ui version="4.0" >
 <class>OptMidi</class>
 <widget class="QWidget" name="OptMidi" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>368</width>
    <height>235</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>OptMidiUI</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="margin" >
    <number>9</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" >
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Queue delay</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="queue_delay" >
       <property name="maximum" >
        <number>5000</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_8" >
       <property name="text" >
        <string>ms</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Send batch period</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="batch_period" >
       <property name="maximum" >
        <number>500</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_9" >
       <property name="text" >
        <string>ms</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QLabel" name="label_6" >
       <property name="text" >
        <string>Keep-In-Touch period</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="in_touch_period" />
     </item>
     <item>
      <widget class="QLabel" name="label_12" >
       <property name="text" >
        <string>s</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_7" >
       <property name="text" >
        <string>Xmpp reconnect interval</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="reconnect_interval" />
     </item>
     <item>
      <widget class="QLabel" name="label_11" >
       <property name="text" >
        <string>s</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox" >
     <property name="text" >
      <string>Allow incoming midi</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_2" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout" >
     <item row="0" column="2" >
      <widget class="QLineEdit" name="server_jid" />
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Server JID</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_4" >
       <property name="text" >
        <string>xmpp-host</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2" >
      <widget class="QLineEdit" name="xmpp_host" />
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="label_5" >
       <property name="text" >
        <string>xmpp-port</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2" >
      <widget class="QLineEdit" name="xmpp_port" >
       <property name="maxLength" >
        <number>4</number>
       </property>
       <property name="fixedWidth" >
        <number>40</number>
       </property>
      </widget>
     </item>
     <item row="0" column="3" >
      <spacer name="horizontalSpacer_4" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" >
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton" >
       <property name="text" >
        <string>Restore</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_3" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
